### ISSUE TITLE
SQL query analysis

### ISSUE THREAD

**Issue by msk-** [mojaloop/finance-portal-backend-service#11]
This PR makes explicit a behaviour that was previously implicit: https://github.com/mojaloop/finance-portal-backend-service/pull/10

Each of the queries modified in that PR should be evaluated for correctness and modified accordingly. As the functionality we've observed seems correct, it's likely that the queries are working correctly, however we should attempt to eliminate usage of `ANY_VALUE` by correct usage of `GROUP BY`, aggregate functions such as `SUM`, or introduction of explicit one-to-one relationships between the variables in the `SELECT` clause to prevent or minimise hard-to-debug issues in future.

**Comment by KamuelaFranco** [comment]
@msk- Do those merges mean this can be closed?

**Comment by msk-** [comment]
No, not yet, I'm not aware that anyone has actually done a decent analysis of the usage of `ANY_VALUE` in the db queries. I somewhat expect that all usages thereof should disappear, but I'm a bit hesitant to declare as much.
